css-layout-classes/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-two-third">
+          <img class="character" src="images/link.png" alt="Link" width="100%">
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best installment
+            in the series. It received praise for its large amount of content and
+            fine-tuning of existing Smash gameplay elements, although its online
+            mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/yoshi.png" alt="Yoshi" width="100%">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-two-third">
+          <img class="character" src="images/kirby.png" alt="Kirby" width="100%">
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/fox.png" alt="Fox" width="100%">
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/pikachu.png" alt="Pikachu" width="100%">
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/ness.png" alt="Ness" width="100%">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third">
+          <h2>Donkey Kong</h2>
+          <p>
+            Donkey Kong is a video game series created by Shigeru Miyamoto. It stars an ape named Donkey Kong who, along with his
+            clan of other apes and monkeys defeats a variety of anthropomorphic enemies. Mainly against the Kremlings, a clan of
+            crocodiles led by King K. Rool that usually steals their banana hoard. The franchise consists mainly of platforming
+            games, as well as spin-offs of various genres. <br> <br>
+
+            The first game was the 1981 arcade game Donkey Kong, who featured the homonymous character as the main antagonist in an
+            industrial construction setting.
+          </p>
+        </div>
+        <div class="column-full">
```

Comment:
this should be `column-one-third` since Donkey Kong should be taking up 1/3 of the row